src/mixer.cc

```diff
@@ -96,6 +96,10 @@ void Mixer::EnterNotify() {
 }
 
 void Mixer::Tick() {
+  cyclus::toolkit::RecordTimeSeries<double>(mixer_record_demand, this, 
+                                            throughput);
```

Comment:
Shouldn't this be `std::min(throughput / mixing_ratios[indx], in_buf_sizes[indx])` for each stream?

Reply:
At the moment we do not support vectors in the RecordTimeSeries, it's fixed to doubles or strings at the moment I believe. Basically RecordTimeSeries uses the first parameter to indicate what to name the column in the cyclus output database. Making it a vector would require a rewrite of the output database I think?

You're welcome to do that if you'd like 😛 .

Reply:
Or just a function in the cyclus backend that take a RecordTimeSeries(vector) and turn it into a bunch of individual ones.

Reply:
The other option would be to make a for loop in the facility that loops through individual isotopes and records those.